samples/sensor/tdk_apex/VERSION

```diff
@@ -0,0 +1,5 @@
+VERSION_MAJOR = 1
+VERSION_MINOR = 0
+PATCHLEVEL = 0
+VERSION_TWEAK = 0
+EXTRAVERSION =
```

Comment:
??

Reply:
I removed the VERSION files. I though these VERSION files were recommended, but I misunderstood the guidelines.